nens_auth_client/tests/test_authorize_view.py

```diff
@@ -55,7 +55,6 @@ def test_authorize(
     qs = parse_qs(token_request.text)
     assert qs["grant_type"] == ["authorization_code"]
     assert qs["code"] == ["code"]
```

Comment:
I researched this point a bit; it turned out that the "state" was not a necessary query parameter. Apparently a bug that got fixed in Authlib.